unittest/Makefile.am

```diff
@@ -0,0 +1,62 @@
+AUTOMAKE_OPTIONS = subdir-objects
+
+AM_CPPFLAGS += \
```

Comment:
Are all those parts of `AM_CPPFLAGS` necessary, or did you add them because they might be needed by future tests? Personally I prefer a little different syntax because I think it is easier to maintain:

    AM_CPPFLAGS += -DUSE_STD_NAMESPACE -DPANGO_ENABLE_ENGINE
    AM_CPPFLAGS += -I$(top_srcdir)/ccmain
    AM_CPPFLAGS += -I$(top_srcdir)/api
    # ...